### PR TITLE
Manolo tlc ciel d8 8_6

### PR DESCRIPTION
**Pull request by nolo16**
image

### CODE REVIEW

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 1]
```diff
@@ -474,28 +476,28 @@ Carefully. Carefully.
 [sha:073f8580b3a9fff379d79fa195c3b14076f1d7b5]{
 -- Page 532, Offset 19622.
 -- 一夜で街を支配しておきながら、人間をひとりも逃がさず、かといって<無|・><駄|・>には殺さず、丹念に上質にわたしの城を築き上げた。
-Taking over the town during the night, letting not a single human escape, though still not killing any %{i}wastefully%{/i}, I built up my caste, meticulously and imperially.
+I took the town over in one night. While letting a single human escape, I avoided needless killing and painstakingly built a castle of the finest caliber.
```
Comment (yk1512) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:479]: forgot to change the 'while' to 'without'

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 2]
```diff
@@ -36,22 +36,23 @@ I remove my glasses, place them by the pillows and close my eyes.
...
 -- Page 504, Offset 19535.
 -- 何か、考え事があった気がするんだけど……」
-I have the impression that something's weighing up on my mind, but..."
+I feel like there was something weighing on my mind."
 }
 [sha:5cdc68d3a3bb8bb383c7b3d5b09c60ad4ac23987]{
 -- Page 505, Offset 19536.
 -- 今夜も眠気に負けてしまう。
-Tonight, I'll let myself into the arms of Morpheus.
+// i read this as morbius
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:44]: [morpheus borpheus morpheus walruses](https://www.youtube.com/watch?v=r9B7BQWnZJs)

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 3]
```diff
@@ -166,17 +167,17 @@ Just a little more and I would remember my own name―――
 [sha:d373bcc76f8c989eff13ce1e1adc6ccf24800e66]{
 -- Page 513, Offset 19561.
 -- 喉からでそうな悲鳴と泣き出しそうな顔を両手で覆って、その場から逃げ去った。
-With a scream that seemed to come out of my throat, I would cover with my hands a face that seemed to be weeping and run away.
+As screams and tears threatened to burst out, I'd cover my face with my hands and run away.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:170]: burst forth

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 4]
```diff
@@ -281,33 +282,33 @@ I had no choice but to spend my days without seeing anyone, without doing anythi
...
 -- Page 521, Offset 19585.
 -- ……けれど、それは間違いだった。
-However, I thought wrongly.
+But I was wrong.
 }
 [sha:eda0ec3b7208a84a1edb799c33e3231e34af9a98]{
 -- Page 521, Offset 19586.
 -- わたしはもっと神さまを信じて、自らの<性|さが>を恐れず、神父さまに相談するべきだったのだ。
-I should have trusted in God more and sought counsel with %{i}Monsieur l'abbe%{/i}, without fearing my own nature.
+// i know this is in france and all but i cant say im for having to deepl shit in the middle of reading
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:295]: i'm with you. this is unnecessary french

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 5]
```diff
@@ -453,17 +454,18 @@ But a truly superior body, an inherent gemstone, had to be the Lord's work.
...
 -- Page 530, Offset 19618.
 -- こと■回目の転生にして、私はようやく、その真実にたどり着いた。
-After having reincarnated ■ times, I finally reached that truth.
+After reincarnating ■ times, I finally reached that truth.
 }
 [sha:ea2d9bf95aa65d21c3dc460fb2c9dfe16902fe07]{
 -- Page 531, Offset 19619.
 -- 目覚めた私は、ごく自然に、水が地面を浸食していくように、ひそやかに街を死都に変えていった。
-Upon this awakening and as naturally as water erodes the ground, I secretly transformed this city into a dead city.
+// i went with ghost town but maybe something like town of death would be better
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:462]: no, this is right. 死都 is not a necropolis, its just an empty place devoid of life

**Review by Avuxo — APPROVED** [review]

**Pull request merged**